### DISCUSSION TITLE
[v2] 视频播放页面下方的分享不见了...

### DISCUSSION THREAD

**Discussion by CKylinMC** [the1812/Bilibili-Evolved#2428] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/11648014/138586425-f05269d3-b8ce-4b4f-9c43-8af93412c023.png)

一键三连右边应该有一个分享按钮的，好像从前几天开始就不见了，我不能确定是因为什么。

不过在我开隐私模式窗口后又能看到那个按钮了...

我把油猴关了，但是那个按钮也没有回来，所以肯定不是哪个脚本干的...

**Comment by the1812** [comment]
虽然看你描述跟脚本没啥关系，但最近修复了外置稍后再看那些，你看看分享还有没有？

**Reply by CKylinMC** [reply]
...刚刚突然找到原因了，是 AdBlock Plus 干的，它自己显示当前页面什么都没屏蔽，但是把它关了之后分享按钮就出来了...

![image](https://user-images.githubusercontent.com/11648014/138902272-1ed2fd8c-86a5-49fb-8cc1-6d68b0caf28c.png)

![image](https://user-images.githubusercontent.com/11648014/138902333-87119fe0-d616-4e04-8cbf-33ea86857bf8.png)
